201-vm-domain-join-existing/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Joins an existing Windows VM to AD Domain",
+  "description": "This template allows you to join a Windows virtual machine into an Active Directory Domain.",
```

Comment:
Would add more detail here...

... allows you to join an existing Windows virtual machine...

The machine must already exist, be in the same resource group (if we leave location as is), etc.  Similar to the readme